apps/addons/buttons.py

```diff
@@ -16,8 +16,16 @@
 @jinja2.contextfunction
 def install_button(context, addon, version=None, show_contrib=True,
                    show_warning=True, src='', collection=None, size='',
-                   detailed=False, mobile=False, impala=False):
-    """If version isn't given, we use the latest version."""
+                   detailed=False, mobile=False, impala=False,
+                   latest_beta=False):
+    """
+    If version isn't given, we use the latest version. You can set latest_beta
+    parameter to use latest beta version instead.
+    """
+    assert not version or not latest_beta, ('version and latest_beta '
+                                            'parameters cannot be specified '
+                                            'at the same time')
```

Comment:
This is fine. I would have written it `assert not (version and latest_beta)`, what do you think?